lib/rules/no-unknown-property.js

```diff
@@ -108,7 +108,7 @@ const SVGDOM_ATTRIBUTE_NAMES = {
 
 const DOM_PROPERTY_NAMES = [
   // Standard
-  'acceptCharset', 'accessKey', 'allowFullScreen', 'allowTransparency', 'autoComplete', 'autoFocus', 'autoPlay',
```

Comment:
In that case, it should be present unless the React version declared in the settings is 16.1+.